packages/dd-trace/src/appsec/index.js

```diff
@@ -105,12 +107,9 @@ function incomingHttpEndTranslator ({ req, res }) {
     payload[addresses.HTTP_INCOMING_PARAMS] = req.params
   }
 
+  // we need to keep this to support other cookie parsers
   if (req.cookies && typeof req.cookies === 'object') {
-    payload[addresses.HTTP_INCOMING_COOKIES] = {}
-
-    for (const k of Object.keys(req.cookies)) {
-      payload[addresses.HTTP_INCOMING_COOKIES][k] = [req.cookies[k]]
-    }
+    payload[addresses.HTTP_INCOMING_COOKIES] = req.cookies
```

Comment:
Should we prevent this lines when the previous cookieParser is already executed? If we detect an attack in cookies, we will detect two attacks in the same request if customer is using cookieParser.

Reply:
An attack shouldn't trigger twice in the waf, so it should be fine